packages/docusaurus-theme-translations/locales/es/theme-common.json

```diff
@@ -3,18 +3,18 @@
   "theme.BackToTopButton.buttonAriaLabel": "Volver al principio",
   "theme.CodeBlock.copied": "Copiado",
   "theme.CodeBlock.copy": "Copiar",
-  "theme.CodeBlock.copyButtonAriaLabel": "Copiar código al portapapeles",
+  "theme.CodeBlock.copyButtonAriaLabel": "Copiar código",
   "theme.CodeBlock.wordWrapToggle": "Alternar ajuste de palabras",
-  "theme.DocSidebarItem.collapseCategoryAriaLabel": "Collapse sidebar category '{label}'",
-  "theme.DocSidebarItem.expandCategoryAriaLabel": "Expand sidebar category '{label}'",
+  "theme.DocSidebarItem.collapseCategoryAriaLabel": "Colapsar categoría de barra lateral '{label}'",
+  "theme.DocSidebarItem.expandCategoryAriaLabel": "Ampliar la categoría de la barra lateral '{label}'",
```

Comment:
I'm not Spanish but that doesn't look good to me. `{label}` is the name of the category, not the name of the sidebar. Wouldn't it be better to put this placeholder closer to the "categoria" word?

Reply:
Not sure it applies to Spanish, but what we have in French:

```
  "theme.DocSidebarItem.collapseCategoryAriaLabel": "Réduire la catégorie '{label}' de la barre latérale",
  "theme.DocSidebarItem.expandCategoryAriaLabel": "Développer la catégorie '{label}' de la barre latérale",
```

Reply:
{Label} is a place holder if i change it when there is a Label it won't show up since is not in the code